src/netius/extra/smtp_a.py

```
@@ -0,0 +1,192 @@
+    def on_serve(self):
+        smtp_r.RelaySMTPServer.on_serve(self)
+        self.activity_url = self.get_env("SMTP_ACTIVITY_URL", self.activity_url)
+        self.activity_secret = self.get_env(
+            "SMTP_ACTIVITY_SECRET", self.activity_secret
+        )
+        if self.activity_url:
+            self.info(
+                "Activity tracking enabled, posting to '%s' ...", self.activity_url
+            )
+
+    def on_relay_smtp(self, smtp_client, connection, froms, tos, contents):
```

Pylint (4.0.5)

[refactor] Too many arguments (6/5)

(R0913)